alacritty/src/config/mod.rs

```diff
@@ -318,6 +319,32 @@ pub fn imports(
     Ok(import_paths)
 }
 
+/// Prune the nulls from the YAML.
+fn prune_yaml_nulls(value: &mut serde_yaml::Value, warn_pruned: bool) {
+    fn walk(value: &mut serde_yaml::Value, warn_pruned: bool) -> bool {
+        if let Some(mapping) = value.as_mapping_mut() {
+            mapping.retain(|key, value| {
+                let retain = !walk(value, warn_pruned);
```

Comment:
This is recursive, right? There doesn't seem a limit to this, so I could probably craft a config that would run into a stack overflow here?

Reply:
Unless the config itself is recursive you should not run into limit? I can limit it just in case if you feel afraid of.